<launch>

  <node pkg="nodelet" type="nodelet" name="runtime_platform_kernel" args="load micros_swarm_framework/RuntimePlatformKernel micros_swarm_framework_manager" output="screen">
    <!--remap from="chatter" to="hello"/-->
  </node>
                 

</launch>

